application/src/test/resources/GetCaseTypesResponseForCCD_TestDefinition_V45.json

```diff
@@ -13,7 +13,8 @@
     "description" : "Content for the Test Jurisdiction.",
     "live_from" : "2017-01-01T00:00:00.000+0000",
     "live_until" : null,
-    "case_types" : [ ]
+    "case_types" : [ ],
+    "banners": []
```

Comment:
In CCD all UI bits of definition are not returned as part of the case type definition. Def Store provides separate endpoints to retrieve the UI definitions under the path `/api/display`. See `DisplayApiController`. Banners is a UI definition, so I think we should add retrieval of banners as a dedicated endpoint in `DisplayApiController`.  Data Store could then retrieve the banners for a certain jur by calling this endpoint.